kubevirt-vmware/pkg/apis/v2v/v1alpha1/ovirtprovider_types.go

```diff
@@ -0,0 +1,80 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OVirtVMDetail contains ovirt vm details as json string
+type OVirtVMDetail struct {
+	// +optional
+	Raw string `json:"raw,omitempty"`
+}
+
+// OVirtVM aligns with maintained UI interface
+type OVirtVM struct {
+	Name string `json:"name"`
+	// +optional
+	DetailRequest bool `json:"detailRequest,omitempty"` // true if details are requested to be loaded
+	// +optional
+	Detail OVirtVMDetail `json:"detail,omitempty"`
+}
+
+// OVirtProviderSpec defines the desired state of OVirtProvider
+type OVirtProviderSpec struct {
+	// +optional
+	Connection string `json:"connection,omitempty"` // name of Secret with ovirt connection details
```

Comment:
https://github.com/oVirt/v2v-conversion-host/pull/75#discussion_r390133132

I think it is better to have the name + namespace here. This would not be a big change on the UI part

Reply:
Both `name` and `namespace are part of [ObjectMeta](https://github.com/ManageIQ/manageiq-v2v-conversion_host/pull/32/files/57354cb1027b42008421a692a357c573d2c89dab#diff-43ed5ee0fb18c0f7b23ae4fe325c440dR60). Is it what you need?

Reply:
We don't need the whole ObjectMeta. Just the name + namespace should be enough to pass for the controller to fetch the right secret

Reply:
It is generated and it should not be changed. I think it is enough for what we need.

Reply:
ok